### PR TITLE
Add fuzz testing intro with link to Rendezvous docs

### PR DESCRIPTION
**Pull request by moodmosaic**

### CODE REVIEW

File: guides-and-tutorials/testing-smart-contracts/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Testing Smart Contracts
+
+Smart contracts are immutable once deployed. Bugs are permanent. Test them
+thoroughly.
+
+This section covers testing Clarity contracts.
+
+* [Fuzz Testing](./fuzz-testing.md): Use Rendezvous to hammer your contract with random
+  inputs. It helps expose edge cases and vulnerabilities.
```
Comment (wileyj) [inline guides-and-tutorials/testing-smart-contracts/README.md:9]: clarunit link

**Review by wileyj — APPROVED** [review]
shipit
